src/components/PlanterDetail.js

```diff
@@ -78,18 +77,17 @@ const PlanterDetail = (props) => {
 
         if (
           !planterRegistration ||
-          planterRegistration.planterId !== planterId
+          planterRegistration[0].planter_id !== planterId
```

Comment:
Can we change the name of the variable to `planterRegistrations` to indicate that it's now an array?